samples/drivers/counter/alarm/src/main.c

```diff
@@ -13,6 +13,15 @@
 #define DELAY 2000000
 #define ALARM_CHANNEL_ID 0
 
+#if defined(DT_RTC_0_NAME)
+#define COUNTER_NAME DT_RTC_0_NAME
+#elif defined(DT_NORDIC_NRF_RTC_RTC_0_LABEL)
+#define COUNTER_NAME DT_NORDIC_NRF_RTC_RTC_0_LABEL
```

Comment:
Can't we do this with an alias in our soc DTSI?

Reply:
added to `dts_fixup.h`

Reply:
`DT_NORDIC_NRF_RTC_RTC_0_LABEL` is actually generated from an alias, such as [this one](https://github.com/zephyrproject-rtos/zephyr/blob/master/dts/arm/nordic/nrf52840.dtsi#L42). You can't (currently) get `DT_RTC_0_NAME` generated by using an alias defined in DTS. The only way is to add such fixups for our SoCs (and although a bit ugly, this seems to be the best way to solve this at the moment).